api/src/main/java/com/okta/sdk/resource/ResourceException.java

```diff
@@ -77,8 +77,9 @@ public int getStatus() {
     }
 
     /**
-     * Get the Okta Error Code
-     * Check http://docs.okta.com/errors/ for the list of Okta Error Codes
+     * Get the Okta Error Code, <a href="https://developer.okta.com/reference/error_codes/index">click here</a> for the
```

Comment:
This page will not render when we move 100% over to CloudFront.

Can you update this to: `/reference/error_codes/`

Reply:
I'm pretty sure I pulled that from a google search, so that might be a bigger issue (but i can update it)

Reply:
**FYI:** With [this change](https://github.com/okta/okta.github.io/pull/1550) we are updating the `sitemap` to use pretty-URL format: `https://site.com/page` instead of `https://site.com/page.html`.